index.html

```diff
@@ -0,0 +1,160 @@
+<!-- <!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    Hello World!
+    <script  src="js/index.js"></script>
+  </body>
+</html> -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+<body>
+	<main id="main">
+
+		<!-- Title of the page -->
+		<h1 id="title">
+			A. P. J. Abdul Kalam
+		</h1>
+
+		<div id="img-div">
+
+			<!--Image of the Tribute Person-->
+			<img src="https://upload.wikimedia.org/wikipedia/commons/6/6e/A._P._J._Abdul_Kalam.jpg" id="image"
+				alt="Error Loading Image">
+			<small id="img-caption">
+				Great Indian scientist and
+				politician who played a leading
+				role in the development of India’s
+				missile and nuclear weapons
+				programs.
+			</small>
+		</div>
```

Comment:
For cleaner code, Please kindly wrap the "h1" tag and "img-div" into "section" tag and change "h1" tag to "header" tag. 👍
Please kindly always use the class attribute for multiple elements (do not use ID selector for multiple elements). You are gonna need to put some effort again and fix everything inside the index.html file, mate. 😮‍💨
It's mentioned here. Please kindly follow all these rules.
https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.